swig/include/python/docs/osr_coordinatetransformation_docs.i

```diff
@@ -77,13 +84,19 @@ list
 
 Examples
 --------
->>> ct.TransformPoints([(-72.58, 44.26), (-72.59, 44.26)])
-[(7390620.052019633, -51202148.77747277, 0.0), (7387261.070131293, -51200373.68798984, 0.0)]
+>>> wgs84 = osr.SpatialReference()
+>>> wgs84.ImportFromEPSG(4326)
+0
+>>> vt_sp = osr.SpatialReference()
+>>> vt_sp.ImportFromEPSG(5646)
+0
+>>> ct = osr.CoordinateTransformation(wgs84, vt_sp)
+>>> ct.TransformPoints([(44.26, -72.58), (44.26, -72.59)])
+[(1619458.1108559777, 641509.1883246159, 0.0), (1616838.2913193079, 641511.9008312856, 0.0)]
 
```

Comment:
I get slightly different result (Proj  9.5.1). Maybe tell in the exaple that coordinates are lat-lon, here I use lon-lat because gdaltransform awaits that.
```
gdaltransform -s_srs epsg:4326 -t_srs epsg:5646
-72.58, 44.26
1619457.99756675 641509.73316664 0
```

Reply:
Added a comment to note that inputs are lat/lon